### ISSUE TITLE
Url with parameter is not recognized

### ISSUE THREAD

**Issue by ptolstoi-sr** [prantlf/grunt-embed-fonts#23]
It recognizes the link with parameters such as into bootstrap-icon url("./fonts/bootstrap-icons.woff2?1fa40e8900654d2863d011707b9fb6f2"). Can You please look at it and repair it if necessary. Thank You in advance.
```
/*!
 * Bootstrap Icons v1.10.5 (https://icons.getbootstrap.com/)
 * Copyright 2019-2023 The Bootstrap Authors
 * Licensed under MIT (https://github.com/twbs/icons/blob/main/LICENSE)
 */

@font-face {
  font-display: block;
  font-family: "bootstrap-icons";
  src: url("./fonts/bootstrap-icons.woff2?1fa40e8900654d2863d011707b9fb6f2") format("woff2"),
url("./fonts/bootstrap-icons.woff?1fa40e8900654d2863d011707b9fb6f2") format("woff");
}
```

**Comment by prantlf** [comment]
Thanks for reporting this! Yes, the path could be followed only by an empty query or just by a hash:

    fonts/file1.eot?#iefix
    fonts/file2.svg#test

I modified the code to recognise also a not empty query without a hash, like `?1fa40e8900654d2863d011707b9fb6f2`, which will be attached after the data URI as the previously supported `?#iefix` was.

The fix was released in [1.1.1](https://github.com/prantlf/grunt-embed-fonts/releases/tag/v1.1.1).
